tests/hazmat/asn1/test_api.py

```diff
@@ -9,7 +9,15 @@
 import cryptography.hazmat.asn1 as asn1
 
 
-class TestClassAPI:
+class TestTypesAPI:
+    def test_repr_printable_string(self) -> None:
+        assert (
+            repr(asn1.PrintableString("MyString"))
+            == "PrintableString(MyString)"
```

Comment:
I think we want `PrintableString("MyString")` (or single quotes) -- using the repr of the str

Reply:
fixed